appveyor.yml

```diff
@@ -16,6 +17,8 @@ clone_depth: 5
 os: Windows Server 2012 R2
 
 environment:
+  CHOCOLATEY_TOKEN:
+    secure: KE3DuBSMhbBxLurkOXLrmAFbF+zptuCJztv1LDq1DfU65A1NSoYTI9JpYE63h/66
```

Comment:
This should be updated if you want to push to chocolatey.org as well. I can add you as collaboratour to the choco package if you like.